src/components/Search/CompareWithBase.tsx

```diff
@@ -302,19 +296,33 @@ function CompareWithBase({
             }
             formIsDisplayed={formIsDisplayed}
           />
+
           <Grid
             item
             xs={2}
-            className={`${dropDownStyles.dropDown} ${bottomStyles.container}`}
+            display='flex'
+            justifyContent={hasEditButton ? 'flex-end' : 'space-between'}
+            className={dropDownStyles.dropDown}
+            alignItems='flex-end'
           >
-            <FrameworkDropdown
-              compact={false}
-              frameworkId={frameWorkId}
-              onChange={(event: SelectChangeEvent) => {
-                const id = +event.target.value as Framework['id'];
-                setframeWorkValue(id);
-              }}
-            />
+            {!hasEditButton && (
+              <SearchFrameworkComponent
+                frameworkId={frameworkId}
+                onChange={(event: SelectChangeEvent) => {
+                  const id = +event.target.value as Framework['id'];
+                  setframeWorkValue(id);
+                }}
+              />
+            )}
+
+            {/**** Hidden Input to capture framework when user updates revisions ****/}
+            {hasEditButton && (
+              <input
+                type='hidden'
+                value={frameworkURL?.toString()}
+                name='framework'
+              ></input>
+            )}
```

Comment:
Sometimes you confuse me, Julien, haha. In one review, you say ternary based renders are complex in react and this case, you say it's preferred. Since it's a nit, I'll leave it as is.

Reply:
It may be my contraryness :o) 
In this specific case I felt like it was better mostly because it's clear it's one or the other, but also because each part is small.
in the end it's really a matter of preference so both work.